### PR TITLE
Adding author bios on pages with included author(s).

### PR DESCRIPTION
**Pull request by weerd**
### What does this PR do?

This PR adds author bios at bottom of a general content page when authors are included. I had forgotten to add them in prior PR that addressed adding authors to the top of the page in #1106.

### Any background context you want to provide?

- [Small Screen](https://user-images.githubusercontent.com/105849/46228636-acd09f00-c330-11e8-82eb-1d904fa5d14e.png)
- [Medium Screen](https://user-images.githubusercontent.com/105849/46228647-b5c17080-c330-11e8-986e-c13aeb1daf5d.png)
- [Large Screen](https://user-images.githubusercontent.com/105849/46228662-c1ad3280-c330-11e8-9388-a43dc92c6cd2.png)

### What are the relevant tickets/cards?

Refs [Pivotal ID #159745156](https://www.pivotaltracker.com/story/show/159745156)

### Checklist

* [x] Documentation added for new features/changed endpoints.
* [ ] Added appropriate feature/unit tests.
* [x] Added screenshot to PR description of related front-end updates on **small** screens.
* [x] Added screenshot to PR description of related front-end updates on **medium** screens.
* [x] Added screenshot to PR description of related front-end updates on **large** screens.

### CODE REVIEW

**Comment by weerd** [comment]
One thing to note regarding the new `AuthorBio` component. I would like to eventually rename `Byline` to `AuthorByline` and combine both into the `Authors/` directory. Seems like a reasonable combo to place into one directory; especially so both can pull from the same default avatar file!

File: resources/assets/components/utilities/Author/AuthorBio.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Figure } from '../../Figure';
+import DEFAULT_AVATAR from './default-avatar.png';
+
+import './author-bio.scss';
+
+const AuthorBio = ({ className, description, jobTitle, name, photo }) => {
+  console.log([name, description, jobTitle, photo]);
```
Comment (mendelB) [inline resources/assets/components/utilities/Author/AuthorBio.js]: 👾

File: resources/assets/components/utilities/Author/author-bio.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@import '../../../scss/next-toolbox';
+
+.author-bio {
+  padding: $half-spacing 0;
```
Comment (mendelB) [inline resources/assets/components/utilities/Author/author-bio.scss]: Total small fry but can you use the `.padding-vertical-md` class to do this? Or is the horizontal `0` rule necessary?

**Comment by mendelB** [comment]
> One thing to note regarding the new `AuthorBio` component. I would like to eventually rename `Byline` to `AuthorByline` and combine both into the `Authors/` directory. Seems like a reasonable combo to place into one directory; especially so both can pull from the same default avatar file!

That sounds wise!

File: resources/assets/components/utilities/Author/AuthorBio.js (hunk 1 above)
Comment (weerd) [reply resources/assets/components/utilities/Author/AuthorBio.js]: dang! nice catch 💃

File: resources/assets/components/utilities/Author/author-bio.scss (hunk 2 above)
Comment (weerd) [reply resources/assets/components/utilities/Author/author-bio.scss]: Noice! Ya, good call. Also reminded me that I wanted to actually put this stuff in a `ul`, so I ended up adding the `.padding-vertical-md` class to the `li`s!

**Pull request merged**